ScnSoft-Education/LeetCodeTraining/PalindromeNumber/PalindromeNumberSolution.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeetCodeTraining.PalindromeNumber
+{
+    //https://leetcode.com/problems/palindrome-number/
+    public static class PalindromeNumberSolution
+    {
+        public static bool IsPalindrome(int x)
```

Comment:
Please improve the code to work with x = 0, because now the code returns false but expected result is true.
Please also update the screenshot with result after this.